src/main/java/uk/gov/hmcts/divorce/common/model/Application.java

```diff
@@ -270,6 +270,18 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateSubmitted;
 
+    @CCD(
+        label = "Is the information provided on the case correct?",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo applicant2ConfirmApplicant1Information;
+
+    @CCD(
+        label = "Explain what is incorrect or needs changing.",
+        access = {DefaultAccess.class}
+    )
+    private String applicant2ExplainsApplicant1IncorrectInformation;
```

Comment:
isn't this field shared between app1 and app2?

Reply:
in what way?
I thought only applicant 2 confirms and applicant 1 is the one doing most of the details.

Reply:
I believe it can go both ways. Lets check with Michael

Reply:
explained how Michael said its different for applicant 1 reviewing it.